### ISSUE TITLE
Contribution guide and/or Code of Conduct?

### ISSUE THREAD

**Issue by Janonard** [rust-dsp/rust-lv2#3]
Right now, it might not be as important to have a contribution guide or a code of conduct, but it may be when looking into the future. I simply like having things sorted out and right now, we have no pressure yet. Any ideas?

**Comment by prokopyl** [comment]
I agree this should be done, however I think we should use a single code of conduct for the whole organization (and same for the contribution guide, minus the crate-specific technical details of course).

Also, while I think having this issue for tracking is good, I think the discussion on what to put in those would be better fitted in the Discourse (probably in different topics even). :)

**Comment by piedoom** [comment]
CoC is something I've been meaning to get around to. I'll post more on forums when I get an opportunity. For now, we should assume the CoC is the same as Rust's own CoC.
